.spelunky2/Locations.py

```diff
@@ -38,38 +38,38 @@ class Spelunky2LocationData(NamedTuple):
     # "Roffy D. Sloth Journal Entry": Spelunky2LocationData(20, "Menu"),
     "Alto Singh Journal Entry": Spelunky2LocationData(21, "Dwelling"),
     "Liz Mutton Journal Entry": Spelunky2LocationData(22, "Jungle"),
-    "Nekka the Eagle Journal Entry": Spelunky2LocationData(23, "Jungle"),
-    "LISE Project Journal Entry": Spelunky2LocationData(24, "Volcana"),
-    "Coco Von Diamonds Journal Entry": Spelunky2LocationData(25, "Volcana"),
+    "Nekka the Eagle Journal Entry": Spelunky2LocationData(23, "Black Market"), # Reduced spawn rate until world 4 is entered
+    "LISE Project Journal Entry": Spelunky2LocationData(24, "Volcana"), # Reduced spawn rate until world 4 is entered

```

Comment:
Are you sure the spawn rate is _reduced_ ? Far as I've ever encountered in the game, these characters are _**dedicated**_ to these areas and literally cannot spawn outside of them.
This is only your commenting so not harmful, just checking your changes to this locations logic is accounting for that